elements/rh-footer/rh-footer-responsive.css.ts

```diff
@@ -92,20 +92,11 @@ export const responsiveStyles = css`
   }
 
   @media screen and (max-width: ${tabletLandscapeBreakpoint}) {
-    /* Add a bit more margin to the primary content on mobile */
-    .main-primary {
-      margin: calc(
-          var(--pf-global--spacer--2xl, 48px) -
-            var(--pf-global--spacer--xl, 32px)
-        )
-        0;
-    }
-
-    .footer-logo {
```

Comment:
Since we are collapsing the space between header and the accordion on mobile, we no longer need this.